rainforest/ml/rfdefinitions.py

```diff
@@ -13,11 +13,17 @@
 # Global imports
 import pickle
 import gzip
+import tempfile
 import numpy as np
 from sklearn.ensemble import RandomForestRegressor
+from sklearn.metrics import r2_score, mean_absolute_error, root_mean_squared_error
 import os
 from scipy.interpolate import UnivariateSpline
 from pathlib import Path
+import mlflow
```

Comment:
should we make mlflow an optional dependency?

for example
```
try:
    import mlflow

    MLFLOW_INSTALLED = True
except ImportError:
   MLFLOW_INSTALLED = False
```

Reply:
good idea, we will change that